src/url.js

```diff
@@ -67,6 +71,21 @@ export function parseUrl(url) {
     search: a.search,
     hash: a.hash,
   };
+
+  // Some IE11 specific polyfills.
+  // 1) IE11 strips out the leading '/' in the pathname.
+  if (info.pathname === '' || info.pathname[0] !== '/') {
+    info.pathname = '/' + info.pathname;
+  }
+
+  // 2) For URLs with implicit ports, IE11 parses to default ports while
+  // other browsers leave the port field empty.
+  if ((info.protocol == 'http:' && info.port == 80)
+      || (info.protocol == 'https:' && info.port == 443)) {
+    info.port = '';
+    info.host = info.hostname;
+  }
+
```

Comment:
The 2 polyfills are to secure the assumptions we made in our code:
1) Default port (80 and 443) are not discarded
2) pathname starts with '/'

Without them, we'd probably still introduce assertions to make sure our assumptions hold. So I think it's harmless to have them here for all browsers.